packages/protocol/scripts/truffle/set_exchange_rate.ts

```diff
@@ -135,7 +136,7 @@ module.exports = async (callback: (error?: any) => number) => {
     // Report it
     await oracles.report(
       stableToken.address,
-      numerator.dividedBy(denominator),
+      toFixed(new BigNumber(numerator).dividedBy(new BigNumber(denominator))),
```

Comment:
No need to make these bignumbers, you should be able to do `toFixed(numerator / denominator)`